### PR TITLE
alphatpx: identified more keys [helwie44], added P30 ROMs and labels [rfka01]

### PR DESCRIPTION
**Pull request by rfka01**

### CODE REVIEW

File: src/mame/drivers/alphatpx.cpp [hunk 1]
```diff
@@ -309,27 +314,28 @@ static INPUT_PORTS_START( alphatp3 )
...
+	PORT_BIT(0x0080, IP_ACTIVE_HIGH, IPT_KEYBOARD) PORT_NAME("F2") PORT_CODE(KEYCODE_F2) PORT_CHAR(UCHAR_MAMEKEY(F2))	// 70h -> 87h func.F2 ok
 
 	PORT_START("COL.15")
 	PORT_BIT(0x0001, IP_ACTIVE_HIGH, IPT_KEYBOARD) PORT_CODE(KEYCODE_8_PAD) PORT_CHAR(UCHAR_MAMEKEY(8_PAD))
 	PORT_BIT(0x0002, IP_ACTIVE_HIGH, IPT_KEYBOARD) PORT_CODE(KEYCODE_5_PAD) PORT_CHAR(UCHAR_MAMEKEY(5_PAD))
 	PORT_BIT(0x0004, IP_ACTIVE_HIGH, IPT_KEYBOARD) PORT_CODE(KEYCODE_3_PAD) PORT_CHAR(UCHAR_MAMEKEY(3_PAD))
-	PORT_BIT(0x0010, IP_ACTIVE_HIGH, IPT_UNKNOWN) // 0x90
-	PORT_BIT(0x0080, IP_ACTIVE_HIGH, IPT_UNKNOWN) // 0xff
+	PORT_BIT(0x0010, IP_ACTIVE_HIGH, IPT_KEYBOARD) PORT_NAME("F1") PORT_CODE(KEYCODE_F1) PORT_CHAR(UCHAR_MAMEKEY(F1))	// 7Dh -> 85H func. F1 ok
+	PORT_BIT(0x0080, IP_ACTIVE_HIGH, IPT_KEYBOARD) PORT_NAME("F11") PORT_CODE(KEYCODE_F11) PORT_CHAR(UCHAR_MAMEKEY(F11)) // SM (typewriter) mode key
```
Comment (cuavas) [inline src/mame/drivers/alphatpx.cpp]: Does the key on the real machine say "F11" on its keycap?  The comment seems to suggest it doesn't.  The `PORT_NAME` macro should always give the name on the key cap on the original system.

Comment (helwie44) [reply src/mame/drivers/alphatpx.cpp]: Ok, I'm going to release the F11 key and the typewriter mode function on e.g. the PC key NUMLOCK to put. Similar to the original P3 / P30 keyboard.

Comment (rfka01) [reply src/mame/drivers/alphatpx.cpp]: Vas, thanks for the explanation. Together with helwie44 I'm working on something more consistent.
Robert

**Comment by etabeta78** [comment]
Just to clarify usual MAME convention: it is more or less ok to map the key
to F11, but PORT_NAME shall reflect the actual marking on the physical
keyboard key.

**Pull request merged**